CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/CVKCanvas.java

```diff
@@ -17,7 +17,7 @@
 
 public class CVKCanvas extends AWTVKCanvas{
     private boolean parentAdded = false;
-    int frameNo = 0;
+    public int frameNo = 0;
```

Comment:
Should this be private and have a function returning it?

Reply:
Atm I think it is just a debugging aide we will probably pull out.  If we formalise it now we might forget to change it later.

Reply:
Actually I've changed it as it probably will stick around.